sdk/storage/Azure.Storage.Queues/api/Azure.Storage.Queues.net8.0.cs

```diff
@@ -42,6 +42,12 @@ public QueueClient(System.Uri queueUri, Azure.Storage.StorageSharedKeyCredential
         public virtual System.Uri GenerateSasUri(Azure.Storage.Sas.QueueSasPermissions permissions, System.DateTimeOffset expiresOn) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public virtual System.Uri GenerateSasUri(Azure.Storage.Sas.QueueSasPermissions permissions, System.DateTimeOffset expiresOn, out string stringToSign) { throw null; }
+        public virtual System.Uri GenerateUserDelegationSasUri(Azure.Storage.Sas.QueueSasBuilder builder, Azure.Storage.Queues.Models.UserDelegationKey userDelegationKey) { throw null; }
```

Comment:
Should we also add new client constructors taking both a Bearer Token and SAS Credential?